src/sections/Projects/Project-grid/projectGrid.style.js

```diff
@@ -275,6 +275,21 @@ export const ProjectWrapper = styled.div`
             }
         }
     }
+
+    
+    .project__card.eleven {
+        grid-column: 7/8;
+        grid-row: 4/6;
+        img {
+            width: 90px;
```

Comment:
Should we use **rem** instead of **px** for best practices?

Reply:
I always use "rem" for measurements, but I noticed "px" in the CSS, so I started using "px" like others do. However, if you change it here, you may have to change it everywhere else as well, right?

Reply:
Yes. Let someone with more experience decide.